### ISSUE TITLE
What is TXOutputs?

### ISSUE THREAD

**Issue by ZhangxyCC** [Jeiwan/blockchain_go#57]
In line 157 blockchain.go, it seems that TXOutputs has never been declared, is it TXOutput[] or I miss something?

**Comment by r0qs** [comment]
Yes, it is a slice of TXOutput: https://github.com/Jeiwan/blockchain_go/blob/master/transaction_output.go#L36
